src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.js';
+
+const ChatLog = (props) => {
+  const chatComponents = props.entries.map(entry => {
```

Comment:
Learn showed this pattern of creating an array of React elements and then referencing it in a JSX expression, so it's definitely okay to do here, but I should just add that I have not found it to be common pattern in practice - most people would just return the array in the component's return statement, e.g.
```js
return (
  <div>
    {props.entries.map(entry => (
      <ChatEntry
        key={entry.id} 
        id={entry.id}  
        sender={entry.sender}
        body={entry.body}
        liked={entry.liked}
        timeStamp={entry.timeStamp}
        updateLikedCount={props.updateLikedCount}
      />
    )}
  </div>
);
```